Readability.js

```diff
@@ -197,20 +197,29 @@ Readability.prototype = {
       return pathBase + uri;
     }
 
-    function convertRelativeURIs(tagName, propName) {
-      var elems = articleContent.getElementsByTagName(tagName);
-      this._forEachNode(elems, function(elem) {
-        var relativeURI = elem.getAttribute(propName);
-        if (relativeURI != null)
-          elem.setAttribute(propName, toAbsoluteURI(relativeURI));
-      });
-    }
-
-     // Fix links.
-    convertRelativeURIs.call(this, "a", "href");
+    var links = articleContent.getElementsByTagName("a");
+    this._forEachNode(links, function(link) {
+      var href = link.getAttribute("href");
+      if (href) {
+        // Replace links with javascript: URIs with text content, since
+        // they won't work after scripts have been removed from the page.
+        if (href.indexOf("javascript:") === 0) {
+          var text = new Text();
+          text.textContent = link.textContent;
+          link.parentNode.replaceChild(text, link);
```

Comment:
This seems to be jsdom-only… dunno what is happening there.

Also, while generating readable content using the patch (using JSDOMParser), things like:

```html
<div>
  Lorem <a href="javascript:void(0)">ipsum</a> dolor sit amet
</div>
```

(notice we're using a `div`, not a `p`)

Gives

```html
<div>
    <p style="display: inline;" class="readability-styled">Lorem</p>ipsum
    <p style="display: inline;" class="readability-styled">dolor sit amet Ut enim ad minim veniam, quis nostrud exercitation ullamco
        laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor
        in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla
        pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa
        qui officia deserunt mollit anim id est laborum.</p>
</div>
```